### PR TITLE
Encryption fixes

### PR DESCRIPTION
**Pull request by ianwremmel**

### CODE REVIEW

File: packages/node_modules/@ciscospark/internal-plugin-user/src/user.js [hunk 1]
```diff
@@ -129,7 +129,12 @@ const User = SparkPlugin.extend({
       resource: `users`
     })
       .then((res) => res.body)
-      .then(tap((user) => this.recordUUID(user)));
+      .then(tap((user) => this.recordUUID({
+        id: user.id,
+        // CI endpoints don't use the same user format as actors, so, email may
+        // be in one of a few fields
+        emailAddress: user.email || user.emailAddress
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/internal-plugin-user/src/user.js]: we're having to massing conversation/user fields in the widget too...

File: packages/node_modules/@ciscospark/test-helper-mocha/src/index.js [hunk 2]
```diff
@@ -87,6 +91,15 @@ module.exports = {
     return inFirefox() ? mochaMethod.skip : mochaMethod;
   },
 
+  skipInIE: function skipInIE(mochaMethod) {
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/test-helper-mocha/src/index.js]: Nice!

File: packages/node_modules/@ciscospark/plugin-logger/src/logger.js [hunk 3]
```diff
@@ -128,6 +129,20 @@ const Logger = SparkPlugin.extend({
   },
 
   /**
+   * Determines if the current level allows logs at the speicified level to be
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/plugin-logger/src/logger.js]: `speicified` not fixed here

File: packages/node_modules/@ciscospark/internal-plugin-encryption/README.md [hunk 4]
```diff
@@ -1,3 +1,81 @@
...
+  .then(() => {
+    spark.internal.encryption.kms.createUnboundKeys({count: 1})
+      .then(([key]) => {
+        return spark.internal.encryption.kms.createResource({
+          userIds: [],
+          key
+        })
+      })
+      .then((kro) => {
+
```
Comment (adamweeks) [inline packages/node_modules/@ciscospark/internal-plugin-encryption/README.md]: Maybe some comments as to what the return values for all of these are?

Comment (ianwremmel) [reply packages/node_modules/@ciscospark/internal-plugin-encryption/README.md]: ah crap. i thought that docs thing was done when I pulled it in here. apparently I never finished it...

I think I'll be pulling that out entirely for now.

File: packages/node_modules/@ciscospark/internal-plugin-user/src/user.js (hunk 1 above)
Comment (ianwremmel) [reply packages/node_modules/@ciscospark/internal-plugin-user/src/user.js]: ?

Comment (adamweeks) [reply packages/node_modules/@ciscospark/internal-plugin-user/src/user.js]: wow, total keyboard fail on that. I don't even know what I was trying to say other than we are doing something similar on widgets.

**Pull request merged**